channels/cl.m3u

```diff
@@ -121,8 +121,6 @@ https://unlimited6-cl.dps.live/tv5/tv5.smil/playlist.m3u8
 http://unlimited1-us.dps.live/tvn/tvn.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TVNHD.cl" tvg-name="TVN HD" tvg-country="CL" tvg-language="" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTLN4WfUMrRsD0b04jbG3EnYrnlV4FYPZURRw&usqp=CAU" group-title="",TVN HD
 http://unlimited10-cl.dps.live/tvn/tvn.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="TVOSanVicente.cl" tvg-name="TVO San Vicente" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/fvnO1X9.png" group-title="",TVO San Vicente
-https://unlimited2-cl-movistar.dps.live/tvo/tvo.smil/tvo/livestream1/chunks.m3u8
```

Comment:
Not 24/7

Adaptive link being https://unlimited2-cl-movistar.dps.live/tvo/tvo.smil/playlist.m3u8

Reply:
Confirm. Provided link works.